client/src/components/FootnoteList/FootnoteList.stories.js

```diff
@@ -0,0 +1,40 @@
+import _ from "lodash";
+import React from "react";
+
+import { FootnoteList } from "./FootnoteList";
+
+export default {
+  title: "FootnoteList",
+  component: FootnoteList,
+};
+
+const Template = (args) => <FootnoteList {...args} />;
+
+// not completed yet
+const footnotes = [
+  {
+    text: "text1",
+    year1: "",
+    year2: "",
+    topic_keys: 1,
+    subject: {
+      name: "name1",
+      plural: "plural1",
+    },
+  },
+  {
+    text: "text2",
+    year1: "",
+    year2: "",
+    topic_keys: 2,
+    subject: {
+      name: "name2",
+      plural: "plural2",
+    },
+  },
+];
```

Comment:
Could use a wider range of props. Comment implies this was planned, so I won't butt in too much. Bit complicated because it requires understanding of the infobase's footnote data. Let me know if you want hints.

Reply:
Not entirely sure how to implement year1 and year2 props into the story without it crashing, this will probably require some thorough investigation into all the props on my part.